codespell_lib/_codespell.py

```diff
@@ -232,6 +232,10 @@ def parse_options(args):
                         help='Comma separated list of words to be ignored '
                              'by codespell. Words are case sensitive based on '
                              'how they are written in the dictionary file')
+    parser.add_argument('-P', '--sub-pairs', type=str, metavar='FILE',
+                        help='Custom substitution text file that contains '
```

Comment:
I'm a bit unclear from the help detail what this is for, is it to "fix up" the dictionary to deal with it matching escape sequences? To actually do sed type runs on my codebase or something else?

Reply:
Is this linked to https://github.com/codespell-project/codespell/issues/233 ?